app/views/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.html.slim

```diff
@@ -0,0 +1,12 @@
+- post = @object
+- answer = post.topic.actable
+- course = topic.course
+- host = course.instance.host
+
+- message.subject = t('.subject', course: course.title, topic: answer.question.display_title)
+
+= simple_format(t('.message',
+                  topic: link_to(answer.question.display_title,
+                                 edit_course_assessment_submission_url(course, answer.question.assessment, answer.submission, host: host)),
```

Comment:
Step param is necessary. Will do that and check with the various assessment formats. Yup planning to do code annotations as part of the same issue, but new PR